doc_src/en/First_Steps.xml

```diff
@@ -5,7 +5,7 @@
 <article lang="en" id="first.steps">
   <title>OmegaT</title>
   <section>
-	<title id="first-steps">First steps</title>
+	<link linkend="first-steps"> </link><title id="first-steps">First steps</title>
```

Comment:
First Steps is not a manual page but greeting contents.
It does not have a link

Reply:
Thank you for catching this one.

It was hard enough to batch process all that with all the IDs that were not standardized. I spent most of my time visually checking the result by using a different CSS that made the invisible space appear as a red dot...